src/libraries/Common/tests/TestUtilities/System/IO/FileCleanupTestBase.Windows.cs

```diff
@@ -23,6 +21,9 @@ public abstract partial class BaseSymbolicLinks : FileSystemTest
         // So instead of using TestDirectory directly, we retrieve the real path with proper casing of the initial folder path.
         private unsafe string GetTestDirectoryActualCasing()
         {
+            if (!PlatformDetection.IsWindows)
+                return TestDirectory;
+
```

Comment:
the logic is located in a `.Windows.cs` file, is it needed?

```suggestion
            Assert.True(PlatformDetection.IsWindows);

```